tools/sidekick/config.json

```diff
@@ -2,6 +2,7 @@
   "project": "BACOM-BLOG",
   "editUrlLabel": "Document Authoring",
   "editUrlPattern": "https://da.live/edit#/{{org}}/{{site}}{{pathname}}",
+  "trustedHosts": ["milo.adobe.com", "*--milo--adobecom.hlx.page"],
```

Comment:
Is this restricting the plugin url hosts or something else? Because we don't have any plugins in da-bacom-blog that come from `--milo--adobecom.hlx.page`

Reply:
Almost. 🙂 
We are trying to restrict which hosts are allowed to make calls to the Helix Admin using the sidekick login.
So you could still load plugins from other URLs, they would just not be allowed to use the sidekick login.

> Because we don't have any plugins in da-bacom-blog that come from --milo--adobecom.hlx.page

Thanks for catching that! I rapid fired ~50 PRs for all `adobecom` repos, there could be some minor misconfigurations. 🙂

Reply:
Fixed in https://github.com/adobecom/da-bacom-blog/pull/47/commits/07badb91c700deaf245248d9e45877b01c0a550b